src/bp-activity/bp-activity-filters.php

```diff
@@ -3601,7 +3601,7 @@ function bb_moderation_remove_mention_count( $num_links, $url, $comment ) {
  * @param object $post Post data.
  */
 function bb_cpt_post_title_save( $post_id, $post ) {
-	if ( empty( $post_id ) || empty( buddypress()->blogs ) ) {
+	if ( empty( $post_id ) || empty( buddypress()->blogs ) || ! bp_is_active( 'activity' ) ) {
```

Comment:
@istiaq-bb I'm unsure of the necessity for this condition, as it is already included in the activity filter file. If the activity is disabled, the file won't load. Please check and revert if not needed.